tests/chainerx_tests/unit_tests/routines_tests/test_indexing.py

```diff
@@ -201,3 +203,61 @@ def forward_xp(self, inputs, xp):
         else:
             b = a.take(indices, axis)
         return b,
+
+
+@op_utils.op_test(['native:0', 'cuda:0'])
+@chainer.testing.parameterize_pytest('cond_shape,x_shape,y_shape', [
+    # Same Shapes
+    ((2, 3), (2, 3), (2, 3)),
+    # Broadcast Shapes
+    ((2, 3), (1, 3), (1, 3)),
+    ((2, 3), (2, 1), (1, 3)),
+    ((2, 3), (2, 3), (1, 3)),
+])
+@chainer.testing.parameterize_pytest(
+    'in_dtypes,out_dtype', dtype_utils.result_dtypes_two_arrays
+)
+@chainer.testing.parameterize_pytest(
+    'condition_dtype', chainerx.testing.integral_dtypes)
```

Comment:
How about adding tests for float conditions?
```suggestion
    'condition_dtype', chainerx.testing.all_dtypes)
```

Reply:
Done.